main/src/com/google/refine/expr/functions/ToDate.java

```diff
@@ -202,4 +203,16 @@ public String getParams() {
     public String getReturns() {
         return "date";
     }
+
+    public static boolean isDate(Object o) {
+        return o instanceof OffsetDateTime;
+    }
+
+    public static OffsetDateTime toDate(Date date) {
+        return date.toInstant().atOffset(ZoneOffset.UTC);
+    }
+
+    public static OffsetDateTime toDate(Calendar date) {
+        return date.toInstant().atOffset(ZoneOffset.UTC);
+    }
```

Comment:
Should these things live in this class? This is the class for the GREL function, these methods look more like utility methods used throughout the code, so I would put them in a utility class.

Reply:
The "right" way to do this would probably be to have an OpenRefine `Date` class which encapsulates all the date handling and then the GREL ToDate could call `com.google.refine...Date.parse()` but the date handling is such a mess I didn't want to risk a big refactor when my goal was to get this fixed for 3.4 once I learned you were still considering adding stuff to the release.

The point of putting the methods here was to keep them co-located with at least some of the date handling code, so people couldn't easily change one without the other. The other date handling utility methods are in ParsingUtilities which isn't a great fit either. 

I can create a separate tiny DateUtility class to hold these for the time being.

Reply:
I decided having them colocated with the rest of the date handling was more important than the inappropriate name, so moved the to `ParsingUtilities`. Investigating the build failures now.